py/picca/tests/delta_extraction/abstract_test.py

```diff
@@ -145,6 +145,8 @@ def compare_fits(self, orig_file, new_file):
                 new_header = new_hdul[hdu_name].header
                 for key in orig_header:
                     self.assertTrue(key in new_header)
+                    if key == 'NAXIS2':
```

Comment:
why do we expect NAXIS2 to have issues? Why do we need to treat it in a special way?

Reply:
array size is different. I'm replacing test files with the new ones, so it will be removed